wca-regulations.md

```diff
@@ -395,7 +395,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - A4) Starting the solve:
     - A4b) The competitor uses their fingers to touch the elevated sensor surfaces of the timer. The competitor's palms must be facing down, and located on the side of the timer that is closer to the competitor. Penalty: time penalty (+2 seconds).
         - A4b1) The competitor must have no physical contact with the puzzle between the inspection period and the start of the solve. Penalty: time penalty (+2 seconds).
-    - A4d) If a Stackmat timer is in use, the competitor starts the solve by confirming that the timer light is green and then removing their hands from the timer (thus starting the timer).
+    - A4d) If a Stackmat timer is in use, the competitor should keep their hands on the timer until they see that a green timer light. They start the solve by removing their hands from the timer (thus starting the timer).
```

Comment:
This doesn't sound like the usual "should" we have.
Also, maybe because I'm not a native speaker, but the bit "...they see that a green timer light" sounds a bit weird.
Would it be "lights up"? Or "...that the timer shows a green light"?

Reply:
@pedrosino Are you arguing "should" needs to be changed to a "must"? That would imply that the competitor cannot lift his hands off before the green light and continue inspection.

Reply:
But that's precisely what we're trying to say, no?
You can't put your hands, take them off, and then put them on again and start the timer. Judge should stop the stopwatch once the competitor lifts their hands, no matter if the timer actually started or not.

Reply:
I thought this was always allowed in case of malfunctions or if timer is not reset/turned on.

Reply:
It's not easy for the judge to see if the light is on or not...
IMO it should not be explicitly permitted in the regulations to do that...it will only cause confusion.

Reply:
`until they see that a green timer light` seems to be wrong.
If `light` is a noun, there shouldn't be a `that`. If `light` is a verb, it seems to be the wrong tense and we don't have green timers. 😛 
If the sentence can be understood in a different way, it also should be changed as I couldn't find this way after reading it ten times. 😄

Reply:
The judge can always look at the display if it's available. Or just wait for a move because of the timer didn't start it's an automatic DNF.

Reply:
Thanks for noticing the grammar issue, folks! Please note that I actually already corrected that in the version of the commit that landed: https://github.com/thewca/wca-regulations/commit/fff14fe406fbe42b4f034d6ffe2df35462f3f994